source/Stargate-HealthCheck/HealthCheckPlugin.class.st

```diff
@@ -0,0 +1,75 @@
+Class {
+	#name : #HealthCheckPlugin,
+	#superclass : #OperationalPlugin,
+	#instVars : [
+		'healthChecks'
+	],
+	#category : #'Stargate-HealthCheck'
+}
+
+{ #category : #'instance creation' }
+HealthCheckPlugin class >> configuredBy: configuration [
+
+	^ self new
+]
+
+{ #category : #accessing }
+HealthCheckPlugin class >> endpoint [
+
+	^ 'health-check'
+]
+
+{ #category : #'instance creation' }
+HealthCheckPlugin class >> new [
```

Comment:
Do we really need an empty check? If a user of this feature does not want any check, they wouldn't use it.

Reply:
It's ok to no have an empty check. Because even if you don't run anything specific just getting to this point confirms that the API es responding on this endpoint and processing requests. So it's like there's always an implicit check performed.